assignments/lambda-classes.js

```diff
@@ -1 +1,188 @@
 // CODE here for your Lambda Classes
+
+// `lambda-classes` - We need a roster of Lambda School personnel.Build it!
+
+//     * We have a school to build here! This project will get you used to thinking about classes in JavaScript and building them from a brand new data set.
+// * Lambda personnel can be broken down into three different types of`people`.
+//   * ** Instructors ** - extensions of Person
+//     * ** Students ** - extensions of Person
+//         * ** Project Managers ** - extensions of Instructors
+//             * ** IMPORTANT ** - You'll need to create 2 - 3 objects for each class and test them according to their unique Attributes. For example:
+
+//                 ```js
+// const fred = new Instructor({
+//   name: 'Fred',
+//   location: 'Bedrock',
+//   age: 37,
+//   favLanguage: 'JavaScript',
+//   specialty: 'Front-end',
+//   catchPhrase: `Don't forget the homies`
+// });
+// ```
+
+// #### Person
+
+// * First we need a Person class. This will be our `base - class`
+// * Person receives `name` `age` `location` all as props
+// * Person receives `speak` as a method.
+// * This method logs out a phrase `Hello my name is Fred, I am from Bedrock` where `name` and `location` are the object's own props
+
+class Person {
+    constructor(attributes) {
+        this.name = attributes.name;
+        this.age = attributes.age;
+        this.location = attributes.location;
+    }
+    speak() {
+        return `Hello my name is ${this.name}, I am from ${this.location}`;
+    }
+};
+
+// #### Instructor
+
+// * Now that we have a Person as our base class, we'll build our Instructor class.
+// * Instructor uses the same attributes that have been set up by Person
+// * Instructor has the following unique props:
+//   * `specialty` what the Instructor is good at i.e. 'redux'
+//   * `favLanguage` i.e. 'JavaScript, Python, Elm etc.'
+//   * `catchPhrase` i.e. `Don't forget the homies`
+//     * Instructor has the following methods:
+//   * `demo` receives a`subject` string as an argument and logs out the phrase 'Today we are learning about {subject}' where subject is the param passed in.
+//   * `grade` receives a`student` object and a`subject` string as arguments and logs out '{student.name} receives a perfect score on {subject}'
+class Instructor extends Person {
```

Comment:
If you want to you can actually use the same naming conventions for constructor/super across all of your different constructors (like attributes or properties).